cloudformation.properties

```diff
@@ -2,12 +2,18 @@ name=Cloudformation
 category=External Analysers
 descripton=Cloudformation rules for json and yaml, integrates cfn-nag reports.
 homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
-archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5,2.0.7,2.0.8,2.0.9
-publicVersions=2.0.10
+archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5,2.0.7,2.0.8,2.0.9,2.1.18
+publicVersions=2.1.18
 
 defaults.mavenGroupId=com.hack23.sonar
 defaults.mavenArtifactId=sonar-cloudformation-plugin
 
+2.1.18.description=Adds support for checkov. Include rules for terraform, kubernetes and azure templates.
+2.1.18.date=2021-11-09
+2.1.18.sqVersions=[7.9,8.9.*]
```

Comment:
All support for 7.9.* is officially over. This range needs to start at 8.9. And do you really mean to _end_ it at 8.9 & not at least support 9.1?
Also, <checks email archives>, if you want to continue supporting Checkov, CNF_NAG &etc reports if there's no conflict with a double-report of metrics. I guess that would be a different version?

Reply:
Only tested/use it on 8.9 and old LTS. Might work on other versions but need to setup some integration test before I want to say it is supported.